.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,8 +9,8 @@
 ### New Feature Submissions:
 
 1. [ ] Does your submission pass tests?
-2. [ ] Have you formatted your code locally using `cargo fmt` command prior to submission?
-3. [ ] Have you checked your code using `cargo clippy` command?
+2. [ ] Have you formatted your code locally using `cargo +nightly fmt --all` command prior to submission?
+3. [ ] Have you checked your code using `cargo +nightly clippy --all --all-features` command?
```

Comment:
we're using stable toolchain for clippy step in github action

Reply:
Addressed.